docs/CODE_STYLE.md

```diff
@@ -1,47 +1,52 @@
 # How to style your code
 
 ## Required:
-- Constructor names should always use PascalCase.\
-That is to prevent any possible overlap with variable names, that is known to cause errors with YYC compiler.
-- As such, variable names, function names, etc, should never use PascalCase.\
-The exception are enum entries.
+- Constructor names should always use `PascalCase`.\
+  This prevents possible overlaps with variable names, which can cause errors with the YYC compiler.
+- Variable names, function names, etc., should never use `PascalCase`.\
+  The exception is for enum entries.
 
 ## Recommended:
 
 ### File names:
-Here it's simple, just follow the general GameMaker way of naming files with type prefixes.
+Follow the general GameMaker convention of using type prefixes in file names.
 
 ### Names:
-All variable names, function names, etc. should use snake_case.\
-Unless stated otherwise, all names should just be `variable_name`, with no additional prefixes.
-- **Local** variables are recommended to have a `_` prefix. This may ease code reading.
+All variable names, function names, etc., should use `snake_case` unless otherwise specified.
+
+- **Local** variables are recommended to have a `_` prefix, which can ease readability.
   - Example: `var _example_e1`.
-- **Global** variables don't require any additional rules, as they already require a `global.` prefix to use.
+- **Global** variables require no additional prefix, as they already use `global.`.
   - Example: `global.example_e1`.
-- **Functions** don't require any prefixes, but to avoid accidental overlaps with instance variables:
-  - Name them with at least 2 words: `draw_something()` and not `drawing()`.
-  - And as an action: `draw_something()` and not `green_apple()`.
-  - Here we just try to follow default GM function naming scheme.
+- **Functions** don’t need prefixes but should:
+  - Use at least two words to avoid overlap with instance variables (e.g., `draw_something()`).
+  - Name functions as actions where possible (`draw_something()` vs. `green_apple()`).
+
 - **Macro** constants:
-  - A short `PREFIX_` should state their group in all caps.
-  - After the prefix, they should be written in snake_case.
-  - Examples: `COL_bright_red`, `COL_dark_green`; `DIR_left`, `DIR_up`.
-  - Here we just try to follow default GM macro naming scheme.
+  - To denote their group, use a short prefix in all caps (e.g., `PREFIX_`).
+  - After the prefix, use `snake_case`.
+  - Examples: `COL_bright_red`, `DIR_left`.
+
 - **Enum** constants:
-  - Name of the enum should start with an `e` prefix and be written in all caps.
-  - Name of the enum entries should use PascalCase.
-  - Example: `enum eCOLORS` with entries `DarkRed`, `Blue`, etc. 
+  - Enum names should start with an `e` prefix and be in all caps.
+  - Enum entries should use PascalCase.
```

Comment:
You can wrap PascalCase here in ` ` as well.

Reply:
```suggestion
  - Enum names should start with an `e` prefix and be in all caps.
  - Enum entries should use `PascalCase`.
```